pkg/ingester/ingester_v2.go

```diff
@@ -1479,6 +1516,7 @@ func (i *Ingester) createTSDB(userID string) (*userTSDB, error) {
 		WALSegmentSize:            i.cfg.BlocksStorageConfig.TSDB.WALSegmentSizeBytes,
 		SeriesLifecycleCallback:   userDB,
 		BlocksToDelete:            userDB.blocksToDelete,
+		MaxExemplars:              i.cfg.BlocksStorageConfig.TSDB.MaxExemplars,
```

Comment:
This is the only value we need to pass to enable the in-memory storage within TSDB, which we already have via vendoring.